app/src/main/java/io/scalac/degree/android/fragment/track/TracksListFragment.java

```diff
@@ -0,0 +1,62 @@
+package io.scalac.degree.android.fragment.track;
+
+import com.annimon.stream.Collectors;
+import com.annimon.stream.Stream;
+import com.annimon.stream.function.Predicate;
+
+import org.androidannotations.annotations.AfterInject;
+import org.androidannotations.annotations.Bean;
+import org.androidannotations.annotations.EFragment;
+import org.androidannotations.annotations.FragmentArg;
+import org.androidannotations.annotations.ViewById;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+
+import java.util.List;
+
+import io.scalac.degree.android.adapter.TracksAdapter;
+import io.scalac.degree.android.fragment.common.BaseListFragment;
+import io.scalac.degree.android.fragment.talk.TalkFragment_;
+import io.scalac.degree.connection.model.SlotApiModel;
+import io.scalac.degree.data.manager.SlotsDataManager;
+import io.scalac.degree33.R;
+
+@EFragment(R.layout.fragment_list)
+public class TracksListFragment extends BaseListFragment {
```

Comment:
wez przenos te pliki tak zeby GH to ogarnal, potem bez sensu kolejny raz te same pliki przegladac